package.json

```diff
@@ -10,6 +10,7 @@
     "preview": "vite preview"
   },
   "dependencies": {
+    "@ant-design/icons": "^5.2.6",
```

Comment:
refer @Parasuramandckap

Reply:
uninstalll this package

Reply:
I have uninstalled ant design and written normal input code